src/test/regress/expected/coordinator_shouldhaveshards.out

```diff
@@ -375,13 +375,47 @@ NOTICE:  executing the command locally: SELECT a.count, b.x, b.y, c.a, c.b, d.co
    100 | 3 | 2 | 3 | 2 |     1
 (1 row)
 
+-- issue 3802
+SET citus.shard_replication_factor TO 2;
+CREATE TABLE dist_table(a int);
+ERROR:  relation "dist_table" already exists
+SELECT create_distributed_table('dist_table', 'a');
+NOTICE:  Copying data from local table...
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$coordinator_shouldhaveshards.dist_table$$)
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+BEGIN;
+-- use transtion block so that we will call ExtractLocalAndRemoteTasks
+INSERT INTO dist_table SELECT a + 1 FROM dist_table;
+ROLLBACK;
+BEGIN;
+SET citus.shard_replication_factor TO 2;
```

Comment:
we already set `shard_replication_factor` to 2 in a few lines above

Reply:
I have actually intentionally used it again here because if someone later adds a test that changes replication factor to 1, this test won't be doing what it needs to do. In this test the replication factor specifically needs to be 2, hence I put it here explicitly. But I am okay with removing it as well hoping that noone will put something before this test. So you decide :)

Reply:
> hoping that noone will put something before this test

actually that makes sense, seting it to 2 again would be safer than hoping 😄